pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,886 @@
+
+###### What to do
+1. Search your codebase for any instances of **the old theme**
+2. Replace it with **the new setting**
+
+##### <span class="usa-tag bg-accent-cool-darker">Markup</span> We updated `usa-footer__logo-heading` to use a p instead of an h3. We improved the accessibility of the footer by converting a non-semantic heading into paragraph text.
+
+###### What to do
+1. TK
+</div>
+<!-- End 2.10.1 section --> 
+</div>
+<!-- End compiler accordion --> 
+
+### 3. Install the USWDS 3.0 package
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+1. In a terminal window, navigate to your project root. 
+2. Install USWDS 3.0 with `npm install @uswds/uswds --save-exact`
+3. Uninstall USWDS 2 with `npm uninstall uswds`
+
+### 4. Update your Sass compiler settings and recompile CSS
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+The location of the USWDS source files is different in USWDS 3.0. You'll need to update your compiler settings to compile your Sass from USWDS 3.0.
+
+<!--Start compiler accordion -->
+<div class="usa-accordion usa-accordion--bordered">
+
+<!-- Start USWDS Gulp section -->
+<h4 class="usa-accordion__heading">
+  <button class="usa-accordion__button" aria-controls="m-a6">
+    If you're using USWDS Gulp
+  </button>
+</h4>
+<div id="m-a6" class="usa-accordion__content site-prose" markdown="1">
+
+1. Search for a line like `const USWDS = "node_modules/uswds/dist"` in your Gulp setup. This indicates that you're using the Gulp setup we distributed as USWDS Gulp.
+1. Update the USWDS `const` to the updated uswds package location
+
+    **Old code:**
+
+    {:.site-terminal}
+    ```js
+    const USWDS = "node_modules/uswds/dist";
+    ```
+
+    **New code:**
+
+    {:.site-terminal}
+    ```js
+    const USWDS = "node_modules/@uswds/uswds";
+    ```
+1. Search for `includePaths` in your project's Gulp files. The paths in this list are where the Sass compiler looks for your source files. In USWDS 3.0
+
+    **Old code:**
+
+    {:.site-terminal}
+    ```js
+    .pipe(
+      sass({
+        includePaths: [
+          PROJECT_SASS_SRC,
+          `${USWDS}/scss`,
+          `${USWDS}/scss/packages`,
+        ],
+      })
+    )
+    ```
+
+    **New code:**
+
+    {:.site-terminal}
+     ```js
+     .pipe(
+      sass({
+        includePaths: [
+          PROJECT_SASS_SRC,
+          `${USWDS},
+          `${USWDS}/packages`,
+        ],
+      })
+    )
+    ```
+1. Recompile your Sass as usual. When it compiles, it is now using USWDS 3.0!
+</div>
+<!-- End USWDS Gulp section --> 
+
+<!-- Start USWDS Compile section -->
+<h4 class="usa-accordion__heading">
+  <button class="usa-accordion__button" aria-controls="m-a7">
+    If you're using USWDS Compile
+  </button>
+</h4>
+<div id="m-a7" class="usa-accordion__content site-prose" markdown="1">
+
+{:.site-terminal}
+```js
+/* gulpfile.js */
+
+const uswds = require("@uswds/compile");
+
+/**
+ * USWDS version
+ */
+
+uswds.settings.version = 3;
+```
```

Comment:
@thisisdano @mejiaj Here's a first pass. Working on standardizing the instruction format to match USWDS Gulp 

1. Search for `const uswds = require("@uswds/compile");` in your Gulp setup. This indicates that you're using the Gulp setup we distributed as USWDS Compile.
2. Update your `gulpfile.js` to file to include `uswds.settings.version = 3;`. This setting manages the default USWDS source paths and will point Gulp to the new USWDS 3.0 package locations.
   ```js
   /* gulpfile.js */

   const uswds = require("@uswds/compile");

   /**
    * USWDS version
    */

   uswds.settings.version = 3;
   ```
3. Recompile your Sass as usual and check that your files compiled as expected. When it compiles successfully, it is now using USWDS 3.0!

Reply:
Done!